tracer/src/Datadog.Trace.Tools.Runner/AnalyzeInstrumentationErrorsCommand.cs

```diff
@@ -0,0 +1,144 @@
+// <copyright file="AnalyzeInstrumentationErrorsCommand.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.InstrumentedAssemblyGenerator;
+using Datadog.InstrumentedAssemblyVerification;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tools.Runner.Checks;
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+namespace Datadog.Trace.Tools.Runner;
+
+internal class AnalyzeInstrumentationErrorsCommand : AsyncCommand<AnalyzeInstrumentationErrorsSettings>
+{
+    public override async Task<int> ExecuteAsync(CommandContext context, AnalyzeInstrumentationErrorsSettings settings)
+    {
+        AnsiConsole.Record();
```

Comment:
```suggestion
```
This was only needed for the instrumentation-telemetry piece, which will be added in a separate PR.

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1